### ISSUE TITLE
[QA] Text on the tooltip in “Extra native gas” section is not fully visible on small screen sizes

### ISSUE THREAD

**Issue by tsadovska** [wormhole-foundation/wormhole-connect#298]
#### Steps:
1. Connect Metamask wallets via WalletConnect in ‘From’ and ’To’ sections
2. Prepare to send CELO from Alfajores to Mumbai
3. Type Amount
4. Move slider to the right most position
5. Observe text on tooltip
6. Move slider to the left most position
7. Observe text on tooltip

#### Actual result:
Text on the tooltip is not fully visible.

#### Expected result:
Text on the tooltip is well displayed. 

#### Tested devices:
* Samsung S20 Android 12, Chrome Version: 111.0.5563.115
*  iPhone 11 iOS 15.4.1, Chrome Version 111.0.5563.72

#### Attachments:
* Slider is moved to the right most position
![Screenshot 2023-04-05 at 16 49 28](https://user-images.githubusercontent.com/10746654/230102359-f0f3bb76-b66b-44a2-a08a-fd0f2744b474.png)
* Slider is moved to the left most position
![Screenshot 2023-04-05 at 16 51 06](https://user-images.githubusercontent.com/10746654/230102772-3d6d49fc-d374-49c5-adfd-d9abebed3f76.png)

**Comment by tsadovska** [comment]
Issue will be rechecked after #747 is resolved.

**Comment by AigaD** [comment]
Still reproducible, tooltip text does not fit to the screen
Tested from: https://wormhole-connect.netlify.app/
Environment: iPhone 14, iOS 16, Safari
Attachment:
![IMG-0121](https://github.com/wormhole-foundation/wormhole-connect/assets/91047061/0cedc08c-b56b-4a37-a94e-1885d5b6065e)
![IMG-0120](https://github.com/wormhole-foundation/wormhole-connect/assets/91047061/d7595ccb-3ad3-40b2-9338-28cd552c356e)

**Comment by agodlevska** [comment]
Issue no longer reproducible on https://wormhole-connect.netlify.app/ and https://wormhole-connect.netlify.app/. 
Now the tooltip does not have text but percentages that fit completely on the screen. 

The issue is reproduced on https://portalbridge.com/, tooltip text does not fit to the screen.

Attachments:
![Screenshot 2024-07-31 at 10 04 37](https://github.com/user-attachments/assets/f80e253c-efc7-4434-b1bf-184e3d907327)
![Screenshot 2024-07-31 at 10 05 13](https://github.com/user-attachments/assets/ac0495ce-7771-48d0-8036-a089d19917ed)

**Comment by tsadovska** [comment]
Issue is no longer relevant after redesign. Tooltip has percentages that fit on the screen.
Closing the ticket.
